### ISSUE TITLE
Error with rollup and version 13.0.0

### ISSUE THREAD

**Issue by artecoop** [jaydenseric/apollo-upload-client#186]
Hello and thanks for this fantastic package.

Using svelte and rollup, after updating from 12.1.0 to 13.0.0 I got this error compiling
`[!] (plugin svelte) Error: Package subpath './package.json' is not defined by "exports" in /home/xxx/xxx/admin/node_modules/extract-files/package.json`

Rolling back to 12.1.0 fixes the issue

Environment:
Svelte 3.20.1
rollup 2.3.2
node 13.12.0
npm 6.14.4

**Comment by jaydenseric** [comment]
This might not be an issue with `apollo-upload-client`, it looks like something elsewhere in the code being bundled is attempting to import `extract-files/package.json` for some reason, which I'm pretty sure is not allowed anymore as it is not exposed in package `exports`.

**Comment by jaydenseric** [comment]
I created a test Rollup project, and everything bundles and runs fine:

- Node.js v13.12.0
- `rollup@2.3.2` and also `rollup@1.32.1`
- `@rollup/plugin-commonjs@11.0.2`
- `@rollup/plugin-node-resolve@7.1.1`
- `apollo-upload-client@13.0.0`
- `extract-files@8.0.0`

This won't work on modern Node.js versions that support conditional package `exports`:

```js
console.log(require('extract-files/package.json'))
```

You will get an error like this:

```
internal/modules/cjs/loader.js:524
  throw new ERR_PACKAGE_PATH_NOT_EXPORTED(basePath, mappingKey);
  ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in /[redacted]/node_modules/extract-files/package.json
```

Tools that try to read `package.json` files should probably start using the `fs` module instead of `require`, because this will start coming up more often as packages ship `exports` for dual support for ESM/CJS consumers.

In your case I don't know what tool specifically is doing it, but maybe raise and issue there or raise an issue at [`extract-files`](https://github.com/jaydenseric/extract-files) if it comes to light we need to add `package.json` to the exports for `require`.

**Comment by jaydenseric** [comment]
I just released [`extract-files@v8.1.0`](https://github.com/jaydenseric/extract-files/releases/tag/v8.1.0) which supports `require('extract-files/package.json')` again, so this should no longer be an issue for you :)

**Comment by artecoop** [comment]
@jaydenseric I can confirm that, with this update everything works perfectly! 
Many thanks!
